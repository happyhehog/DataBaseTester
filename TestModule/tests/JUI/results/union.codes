{"status":"OK","command":"SELECT * FROM t1 UNION SELECT * FROM t2;","answer":"id|f1       |f2       |\n1 |22.000000|55.000000|\n2 |5.600000 |5.600000 |\n3 |44.000000|50.000000|\n1 |3.400000 |5.600000 |\n3 |9.000000 |99.000000|"}
{"status":"OK","command":"SELECT * FROM t1 UNION SELECT * FROM t2 UNION SELECT f1, f2 FROM t3;","answer":"f1       |f2       |\n77.000000|2.000000 |\n66.000000|2.000000 |\n11.000000|2.000000 |\n1        |22.000000|\n2        |5.600000 |\n3        |44.000000|\n1        |3.400000 |\n3        |9.000000 |"}
{"status":"OK","command":"SELECT id, f2 FROM t2 UNION SELECT id, f2 FROM t3;","answer":"id|f2       |\n1 |2.000000 |\n2 |2.000000 |\n3 |2.000000 |\n1 |55.000000|\n2 |5.600000 |\n3 |50.000000|"}
{"status":"OK","command":"SELECT * FROM t1 UNION SELECT * FROM t2;","answer":"id|f1       |f2       |\n1 |22.000000|55.000000|\n2 |5.600000 |5.600000 |\n3 |44.000000|50.000000|\n1 |3.400000 |5.600000 |\n3 |9.000000 |99.000000|"}
{"status":"OK","command":"SELECT id, f1 FROM t1 UNION SELECT f1 FROM t2 UNION SELECT f2, f1 FROM t3;","answer":"f1       |f2      |\n77.000000|2.000000|\n66.000000|2.000000|\n11.000000|2.000000|"}
{"status":"OK","command":"SELECT * FROM t1 UNION SELECT * FROM t2 UNION SELECT * FROM t3;","answer":"id|f1       |f2       |\n1 |77.000000|2.000000 |\n2 |66.000000|2.000000 |\n3 |11.000000|2.000000 |\n1 |22.000000|55.000000|\n2 |5.600000 |5.600000 |\n3 |44.000000|50.000000|\n1 |3.400000 |5.600000 |\n3 |9.000000 |99.000000|"}
{"status":"OK","command":"SELECT f2 FROM t1 INTERSECT SELECT f2 FROM t2;","answer":"f2      |\n5.600000|\n5.600000|\n5.600000|"}
{"status":"OK","command":"SELECT * FROM t1 INTERSECT SELECT * FROM t2 INTERSECT SELECT * FROM t3;","answer":"id|f1      |f2      |\n4 |4.000000|4.000000|"}
