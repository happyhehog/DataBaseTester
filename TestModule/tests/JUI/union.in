[@PrintLevel] MAIN
[@Clear]

[@NoOutput]
CREATE TABLE t1(id INT, f1 FLOAT, f2 FLOAT);
CREATE TABLE t2(id INT, f1 FLOAT, f2 FLOAT);
CREATE TABLE t3(id INT, f1 FLOAT, f2 FLOAT);

INSERT INTO t1 VALUES(1, 3.4, 5.6);
INSERT INTO t1 VALUES(2, 5.6, 5.6);
INSERT INTO t1 VALUES(3, 9, 99);

INSERT INTO t2 VALUES(1, 22, 55);
INSERT INTO t2 VALUES(2, 5.6, 5.6);
INSERT INTO t2 VALUES(3, 44, 50);

INSERT INTO t3 VALUES(1, 77, 2);
INSERT INTO t3 VALUES(2, 66, 2);
INSERT INTO t3 VALUES(3, 11, 2);
[@NoOutput]

@@ Union
SELECT * FROM t1 UNION SELECT * FROM t2;

[@Print]
SELECT * FROM t1 UNION SELECT * FROM t2 UNION SELECT f1, f2 FROM t3;

[@Print]
SELECT id, f2 FROM t2 UNION SELECT id, f2 FROM t3;

[@Print]
SELECT * FROM t1 UNION SELECT * FROM t2;

[@Print]
SELECT id, f1 FROM t1 UNION SELECT f1 FROM t2 UNION SELECT f2, f1 FROM t3;

[@Print]
SELECT * FROM t1 UNION SELECT * FROM t2 UNION SELECT * FROM t3;

[@Print]
SELECT f2 FROM t1
INTERSECT
SELECT f2 FROM t2;

[@Print]
[@NoOutput]
INSERT INTO t1 VALUES(4, 4, 4);
INSERT INTO t2 VALUES(4, 4, 4);
INSERT INTO t3 VALUES(4, 4, 4);
[@NoOutput]

SELECT * FROM t1 INTERSECT SELECT * FROM t2 INTERSECT SELECT * FROM t3;