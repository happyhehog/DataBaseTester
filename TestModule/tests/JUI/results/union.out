id|f1       |f2       |
1 |22.000000|55.000000|
2 |5.600000 |5.600000 |
3 |44.000000|50.000000|
1 |3.400000 |5.600000 |
3 |9.000000 |99.000000|

f1       |f2       |
77.000000|2.000000 |
66.000000|2.000000 |
11.000000|2.000000 |
1        |22.000000|
2        |5.600000 |
3        |44.000000|
1        |3.400000 |
3        |9.000000 |

id|f2       |
1 |2.000000 |
2 |2.000000 |
3 |2.000000 |
1 |55.000000|
2 |5.600000 |
3 |50.000000|

id|f1       |f2       |
1 |22.000000|55.000000|
2 |5.600000 |5.600000 |
3 |44.000000|50.000000|
1 |3.400000 |5.600000 |
3 |9.000000 |99.000000|

f1       |f2      |
77.000000|2.000000|
66.000000|2.000000|
11.000000|2.000000|

id|f1       |f2       |
1 |77.000000|2.000000 |
2 |66.000000|2.000000 |
3 |11.000000|2.000000 |
1 |22.000000|55.000000|
2 |5.600000 |5.600000 |
3 |44.000000|50.000000|
1 |3.400000 |5.600000 |
3 |9.000000 |99.000000|

f2      |
5.600000|
5.600000|
5.600000|

id|f1      |f2      |
4 |4.000000|4.000000|
